reqs/requirements-dev-clickhouse.txt

```diff
@@ -1,2 +1,4 @@
-clickhouse-sqlalchemy>=0.2.2
+clickhouse-sqlalchemy>=0.2.2; python_version < "3.12"
+clickhouse-sqlalchemy>=0.3.0; python_version >= "3.12" # SQLALchemy 2 support added in 0.3.0
 pandas<2.2.0; python_version >= "3.9"
+pandas>=2.0.0; python_version <= "3.12"
```

Comment:
SQLAlchemy 2.0 support was added
https://github.com/xzkostyan/clickhouse-sqlalchemy/blob/master/CHANGELOG.md#030---2023-11-06

I believe this is why we are restricting the `pandas` version. I think we can lift this restriction of all versions of python.